Source/WebCore/editing/cocoa/HTMLConverter.mm

```diff
@@ -1198,6 +1207,34 @@ static Class _WebMessageDocumentClass()
     return _WebMessageDocumentClass;
 }
 
+#if ENABLE(MULTI_REPRESENTATION_HEIC)
+BOOL HTMLConverter::_addMultiRepresentationHEICAttachmentForImageElement(HTMLImageElement& element)
+{
+    RefPtr image = element.image();
+    if (!image)
+        return NO;
+
+    WebMultiRepresentationHEICAttachment *attachment = image->adapter().multiRepresentationHEIC();
+    if (!attachment)
+        return NO;
+
+    NSUInteger textLength = [_attrStr length];
+
+    RetainPtr<NSString> string = adoptNS([[NSString alloc] initWithFormat:@"%C", static_cast<unichar>(NSAttachmentCharacter)]);
+    NSRange rangeToReplace = NSMakeRange(textLength, 0);
+
+    [_attrStr replaceCharactersInRange:rangeToReplace withString:string.get()];
+    rangeToReplace.length = [string length];
+    if (rangeToReplace.location < _domRangeStartIndex)
+        _domRangeStartIndex += rangeToReplace.length;
+
+    [_attrStr addAttribute:NSAttachmentAttributeName value:attachment range:rangeToReplace];
+
+    _flags.isSoft = NO;
```

Comment:
what do es this mean?

Reply:
It ensures leading spaces are not suppressed after an attachment. It's already used for regular attachments.